<?xml version="1.0" encoding="utf-8"?>
<testsuite package="org.spotbugs" time="0" tests="19" errors="2" name="findbugs">
<testcase time="0" name="view/Console.java" classname="FindBugs Issues">
</testcase><testcase time="0" name="model/Contract.java" classname="FindBugs Issues">
</testcase><testcase time="0" name="model/Registry.java" classname="FindBugs Issues">
</testcase><testcase time="0" name="view/MemberChoice.java" classname="FindBugs Issues">
</testcase><testcase time="0" name="model/Interval.java" classname="FindBugs Issues">
</testcase><testcase time="0" name="model/Member.java" classname="FindBugs Issues">
</testcase><testcase time="0" name="model/Day.java" classname="FindBugs Issues">
</testcase><testcase time="0" name="model/DayCounter.java" classname="FindBugs Issues">
</testcase><testcase time="0" name="App.java" classname="FindBugs Issues">
</testcase><testcase time="0" name="model/Type.java" classname="FindBugs Issues">
</testcase><testcase time="0" name="view/ItemChoice.java" classname="FindBugs Issues">
</testcase><testcase time="0" name="model/Item.java" classname="FindBugs Issues">
<failure message="issues" ><![CDATA[lines: 14-142 
setOwner(Member) may expose internal representation by storing an 
  externally mutable object into Item.owner 
May expose internal representation by incorporating reference to mutable 
  object 
This code stores a reference to an externally mutable object into the 
  internal representation of the object. If instances are accessed by 
  untrusted code, and unchecked changes to the mutable object would 
  compromise security or other important properties, you will need to do 
  something different. Storing a copy of the object is better approach in 
  many situations.
----------------
]]></failure>
</testcase><testcase time="0" name="view/UpdateMemberChoice.java" classname="FindBugs Issues">
</testcase><testcase time="0" name="model/Validator.java" classname="FindBugs Issues">
</testcase><testcase time="0" name="view/MainChoice.java" classname="FindBugs Issues">
</testcase><testcase time="0" name="controller/User.java" classname="FindBugs Issues">
<failure message="issues" ><![CDATA[lines: 23-347 
quitProgram() invokes System.exit(...), which shuts down the entire virtual 
  machine 
Method invokes System.exit(...) 
Invoking System.exit shuts down the entire Java virtual machine. This 
  should only been done when it is appropriate. Such calls make it hard or 
  impossible for your code to be invoked by other code. Consider throwing a 
  RuntimeException instead.
----------------
]]></failure>
</testcase><testcase time="0" name="model/persistence/IfPersistence.java" classname="FindBugs Issues">
</testcase><testcase time="0" name="model/persistence/Persistence.java" classname="FindBugs Issues">
</testcase><testcase time="0" name="model/IdGenerator.java" classname="FindBugs Issues">
</testcase></testsuite>